archetypes.js

```diff
@@ -0,0 +1,1484 @@
+const emptyBuild = {
+  position: 'None',
+  shortPosition: 'None',
+  archetype: 'None',
+  baseTpe: 0,
+  strengthMin: 0,
+  strengthMax: 0,
+  agilityMin: 0,
+  agilityMax: 0,
+  armMin: 0,
+  armMax: 0,
+  intelligenceMin: 0,
+  intelligenceMax: 0,
+  throwingAccuracyMin: 0,
+  throwingAccuracyMax: 0,
+  tacklingMin: 0,
+  tacklingMax: 0,
+  speedMin: 0,
+  speedMax: 0,
+  handsMin: 0,
+  handsMax: 0,
+  passBlockingMin: 0,
+  passBlockingMax: 0,
+  runBlockingMin: 0,
+  runBlockingMax: 0,
+  enduranceMin: 0,
+  enduranceMax: 0,
+  kickPowerMin: 0,
+  kickPowerMax: 0,
+  kickAccuracyMin: 0,
+  kickaccuracyMax: 0
+};
+
+const safetyCenterFielder = {
+  position: 'Safety',
+  shortPosition: 'S',
+  archetype: 'Center Fielder',
+  baseTpe: 466,
+  strengthMin: 40,
+  strengthMax: 70,
+  agilityMin: 70,
+  agilityMax: 95,
+  armMin: 1,
+  armMax: 25,
+  intelligenceMin: 40,
+  intelligenceMax: 80,
+  throwingAccuracyMin: 1,
+  throwingAccuracyMax: 25,
+  tacklingMin: 50,
+  tacklingMax: 75,
+  speedMin: 70,
+  speedMax: 100,
+  handsMin: 55,
+  handsMax: 75,
+  passBlockingMin: 1,
+  passBlockingMax: 25,
+  runBlockingMin: 1,
+  runBlockingMax: 25,
+  enduranceMin: 70,
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const safetyPlayermaker = {
```

Comment:
small thing but should we rename it to safetyPlaymaker instead of safetyPlay**er**maker?

Reply:
Good catch, fixed